packages/ui/src/TextArea.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ChangeEvent, FC, FocusEvent, ReactElement, useRef } from 'react'
+import { v4 as uuid } from 'uuid'
+import cn from 'classnames'
+
+import { DataTestProp } from '@hazelcast/helpers'
+import { Error, errorId } from '../src/Error'
+import { Help } from '../src/Help'
+import { Label } from './Label'
+
+import styles from './TextArea.module.scss'
+
+export type TextAreaCoreProps = {
+  name: string
+  value?: string
+  onBlur?: (e: FocusEvent<HTMLTextAreaElement>) => void
+  onChange: (e: ChangeEvent<HTMLTextAreaElement>) => void
+  error?: string
+}
+
+export type TextAreaExtraProps = {
+  label: string
+  textareaClassName?: string
+  errorClassName?: string
+  resizable?: boolean
+  helperText?: string | ReactElement
+} & Partial<Pick<HTMLTextAreaElement, 'className' | 'disabled' | 'placeholder' | 'required'>>
```

Comment:
Should we also support `rows`? So that we can have control of an initial height?

Reply:
Do you use rows anywhere in the Cloud? I suggest we make our TextArea of fixed height which we can override with a className. WDYT?

Reply:
I think it's nice HTML standard attr that should be allowed, but we're not heavily dependent on it. It's more like `nit`